src/main/java/roomescape/config/WebMvcConfiguration.java

```diff
@@ -0,0 +1,30 @@
+package roomescape.config;
+
+import java.util.List;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class WebMvcConfiguration implements WebMvcConfigurer {
+
+    private final LoginMemberArgumentResolver loginMemberArgumentResolver;
+    private final LoginInterceptor loginInterceptor;
+
+    public WebMvcConfiguration(final LoginMemberArgumentResolver loginMemberArgumentResolver,
+                               final LoginInterceptor loginInterceptor) {
+        this.loginMemberArgumentResolver = loginMemberArgumentResolver;
+        this.loginInterceptor = loginInterceptor;
+    }
+
+    @Override
+    public void addArgumentResolvers(final List<HandlerMethodArgumentResolver> resolvers) {
+        resolvers.add(loginMemberArgumentResolver);
+    }
+
+    @Override
+    public void addInterceptors(final InterceptorRegistry registry) {
+        registry.addInterceptor(loginInterceptor).addPathPatterns("/admin/**");
```

Comment:
```suggestion
        registry.addInterceptor(loginInterceptor)
                .addPathPatterns("/admin/**");
```

위와 같이 개행을 하면 가독성을 높일 수 있을 것 같습니다.

Reply:
아하 개행을 해주면 좋겠네요!